src/TaylorSeries/parameter.jl

```diff
@@ -14,7 +14,7 @@
 These parameters can be changed using [`set_variables`](@ref)
 """
 mutable struct ParamsTaylor
-    order::Int
+    order::Vector{Int}
```

Comment:
I would prefer to pluralize this name `order` -> `orders`, since the field now represents a list of maximum expansion orders. Similarly, pluralize `get_order` -> `get_orders`.